Functions/xASL_quant_BSupCalculation.m

```diff
@@ -88,18 +90,16 @@
 if PresaturationTime
     % Recovery from 0 to 1 at T1 for the period between PresaturationTime and the first BSup pulse
     SignalPercentageInitial(1) = 1 - (1 - 0)*exp(-(BackgroundSuppressionPulseTime(1)-PresaturationTime)/T1Time);
-    PresaturationTimeDummy = PresaturationTime;
 else
     % Without saturation, full signal is preset
     SignalPercentageInitial(1) = 1;
     % For the purpose of visualization, in the absence of presaturation we set the presaturation time to 1 
-    PresaturationTimeDummy = 1;
 end
```

Comment:
I removed PresaturationTimeDummy, since I didn't see where it is used. Hope that was correct. Please re-check 😉

Reply:
Best to ask Jan!